builtin/credential/approle/path_role.go

```diff
@@ -1734,10 +1714,20 @@ func (b *backend) pathRolePoliciesRead(ctx context.Context, req *logical.Request
 		return nil, nil
 	}
 
+	p := role.TokenPolicies
+	if p == nil {
+		p = []string{}
+	}
+	d := map[string]interface{}{
+		"token_policies": p,
+	}
+
+	if len(role.Policies) > 0 {
+		d["policies"] = role.Policies
```

Comment:
Do we also want to add a warning that both are same and that one of them is deprecated?

Reply:
We have a text in path-help